index.js

```diff
@@ -71,12 +62,14 @@ export default class SideMenu extends React.Component {
   constructor(props: Props) {
     super(props);
 
+    const { width, height } = Dimensions.get('window');
```

Comment:
it can be typed using `WindowDimensions`

Reply:
Fixed